modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,65 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline, resource level
+
+By default, policies are applied to the entire API. However, you can implement an additional level of granularity for policies, where access is controlled based on the a criteria, called resource-level policies. 
+
+In this level of granularity, policies are applied to only those request that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
```

Comment:
```suggestion
At the resource level of granularity, policies are applied to only those requests that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
```
Clarify: If all but CORS are resource level, then why state that all are by default applied to the whole API? This sounds contradictory.

Reply:
All policies unless configured otherwise are applied to the entire API. The customer might decide to have more granularity by configuring it defining the resource level. This configuration is not available on CORS.

Reply:
Done! Thanks!